src/components/userFeedback/UserFeedback.spec.tsx

```diff
@@ -0,0 +1,143 @@
+import { shallow, ShallowWrapper } from 'enzyme';
```

Comment:
Both shallow or mount would be fine in this scenario. Shallow tests components 'as unit', and [its methods aren't indirectly asserting on behavior of child components](http://airbnb.io/enzyme/docs/api/shallow.html). Since there is only one html element in the component, it seems to me that both could have been utilized here.